picard/ui/tablebaseddialog.py

```diff
@@ -52,6 +52,7 @@ class ResultTable(QtWidgets.QTableWidget):
 
     def __init__(self, parent):
         super().__init__(parent)
+        self._parent_dialog = parent
```

Comment:
Parent is available via `self.parent()`, so does not need to be stored separately.

Reply:
They differ, `self.parent()` is the parent widget, not the parent dialog